docs/utils/examples/beams2d.py

```diff
@@ -0,0 +1,12 @@
+from typing import Any
+
+import numpy as np
+from numpy.typing import NDArray
+
+from engibench.problems.beams2d import Beams2D
+
+
+def run_job(config: dict[str, Any]) -> NDArray[np.float64]:
```

Comment:
Shouldn't the config be passed to optimize? I see in the example that `volfrac` and `forcedist` are being passed. These are for optimization, not for problem instantiation (at least they used to be configurable only when calling optimize or simulate).